modules/shared/partials/acid-transactions.adoc

```diff
@@ -206,28 +217,29 @@ Note that expiration is not guaranteed to be followed precisely.
 For example, if the application were to do a long blocking operation inside the lambda (which should be avoided), then expiration can only trigger after this finishes.
 Similarly, if the transaction attempts a key-value operation close to the expiration time, and that key-value operation times out, then the expiration time may be exceeded.
 
-=== TransactionCommitAmbiguous
+=== {transaction-commit-ambiguous} 
 
 As discussed <<mechanics,previously>>, each transaction has a 'single point of truth' that is updated atomically to reflect whether it is committed.
 
 However, it is not always possible for the protocol to become 100% certain that the operation was successful, before the transaction expires.
 That is, the operation may have successfully completed on the cluster, or may succeed soon, but the protocol is unable to determine this (whether due to transient network failure or other reason).
 This is important as the transaction may or may not have reached the commit point, e.g. succeeded or failed.
 
-The library raises TransactionCommitAmbiguous to indicate this state.
-It should be rare to receive this exception.
+Couchbase transactions will raise `{transaction-commit-ambiguous}` to indicate this state.
```

Comment:
the code backticks need to be within the variable don't they?

Reply:
Not in all cases, for example we use **{transaction-commit-ambiguous}** as a heading as well, so including the back ticks wouldn't allow us to reuse this variable in other places.

https://github.com/couchbase/docs-sdk-common/pull/46/files#diff-99180d5d8fb133a299c04b915f1fac0c35a880d99bca85864460b707ebb8aa6cR220